sdk/storage/inc/datalake/datalake_utilities.hpp

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "common/crypt.hpp"
+
+#include <map>
+#include <sstream>
+#include <string>
+
+namespace Azure { namespace Storage { namespace DataLake {
+
+  inline std::map<std::string, std::string> ParseMetadataString(
```

Comment:
That function is called `SerializeMetadataXxx`, so I think this should be called `DeserializeXxx`.